src/promptflow/promptflow/executor/_input_assignment_parser.py

```diff
@@ -19,18 +20,44 @@ def parse_value(i: InputAssignment, nodes_outputs: dict, flow_inputs: dict):
     if i.value_type == InputValueType.FLOW_INPUT:
         if i.value not in flow_inputs:
             flow_input_keys = ", ".join(flow_inputs.keys()) if flow_inputs is not None else None
-            raise InputNotFound(message=f"{i.value} is not found from flow inputs '{flow_input_keys}'")
+            raise InputNotFound(
+                message_format=(
+                    "The input '{input_name}' is not found from flow inputs '{flow_input_keys}'. "
```

Comment:
1)  I think you might consider create a template for this? is this case only happen for single node run? 


2) do we have yaml test case for this error ?

Reply:
1. updated.
2. added unit tests